docs/quick-start.md

```
@@ -0,0 +1,149 @@
+# From workspace root (recommended)
+./scripts/xrd-transform.sh -v template-namespace/configuration/xrd.yaml
+
+# Or from plugin directory
+cd plugins/ingestor && \
+cat ../../../template-namespace/configuration/xrd.yaml | \
+npx ts-node --project tsconfig.cli.json src/xrd-transform/cli/xrd-transform-cli.ts -v
+```
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)